src/Commands/ApigeeEdgeCommands.php

```diff
@@ -19,29 +19,43 @@
 
 namespace Drupal\apigee_edge\Commands;
 
+use Consolidation\AnnotatedCommand\CommandData;
 use Drupal\apigee_edge\CliServiceInterface;
+use Drupal\apigee_edge\Util\EdgeConnectionUtilServiceInterface;
 use Drush\Commands\DrushCommands;
 
 /**
  * Drush 9 command file.
  */
 class ApigeeEdgeCommands extends DrushCommands {
 
+  // Default base url.
+  const DEFAULT_BASE_URL = 'https://api.enterprise.apigee.com/v1';
+
+  // Default role name to create in Apigee Edge.
+  const DEFAULT_ROLE_NAME = 'drupalportal';
```

Comment:
This information belongs to the service that takes care of the role creation, not to this base class.

Reply:
Moved to `EdgeConnectionUtilServiceInterface`